tools/daily-dev-build-variable/daily-dev-build-variable.yml

```diff
@@ -0,0 +1,12 @@
+# This script fragment is used across our repos to set a variable "SetDevVersion" which
+# is used when this pipeline is going to be generating and publishign daily dev builds.
+
+steps:
+- pwsh: |
+    $setDailyDevBuild = "false"
+    if (('$(Build.Reason)' -eq 'Schedule') -and ('$(System.TeamProject)' -eq 'internal')) {
+      $setDailyDevBuild = "true"
+    }
+    echo "##vso[task.setvariable variable=SetDevVersion]$setDailyDevBuild"
+  displayName: "Setup Versioning Properties"
+  condition: eq(variables['SetDevVersion'], '')
```

Comment:
Silly question but does this need to have a newline? I'm approving the changes either way but was just curious since the comparison view is flagging that.

Reply:
Should matter in this case, but often people do like new lines at the end of files.